plugins/eddn.py

```diff
@@ -599,21 +602,16 @@ def export_journal_shipyard(self, cmdr: str, is_beta: bool, entry: Mapping[str,
 
         # this.shipyard = (horizons, shipyard)
 
-    def export_journal_entry(self, cmdr: str, is_beta: bool, entry: Mapping[str, Any]) -> None:
+    def export_journal_entry(self, cmdr, entry, msg):
```

Comment:
why remove the type hints on this?

Reply:
This was due to actually renaming this to `export_journal_generic` and making a new `export_journal_entry`, I'll fix shortly.